stl/inc/format

```diff
@@ -840,15 +840,36 @@ public:
     }
 };
 
+// clang-format off
+template<class _Context, class _Ty>
+concept _Has_formatter = requires(const _Ty& _Val, _Context& _Ctx) {
+    _STD declval<typename _Context::template formatter_type<_Ty>>().format(_Val, _Ctx);
+};
+// clang-format on
+
+// See [format.arg]/5

```

Comment:
Because Standardese changes and paragraphs are renumbered over time, we try to cite Working Papers along with section stable names:
```suggestion
// See N4878 [format.arg]/5
```